syb8200/4주차/app/src/main/java/com/yapp/composehomework1/Homework1.kt

```diff
@@ -0,0 +1,340 @@
+            title = "교통카드 잔액보고 충전하기",
+        )
+        Spacer(modifier = Modifier.size(20.dp))
+        ItemSection(
+            icon = Icons.Filled.ThumbUp,
+            color = Color.Green,
+            subTitle = null,
+            title = "토스머니 내역 보기",
+        )
+        Spacer(modifier = Modifier.size(20.dp))
+        ItemSection(
+            icon = Icons.Filled.Favorite,
+            color = Color.Yellow,
+            subTitle = null,
+            title = "토스머니 충전하기",
+        )
+    }
+
+    CardSection(title = "용돈기입장") {
+        Spacer(modifier = Modifier.size(20.dp))
+
+        ItemSection(
+            icon = Icons.Filled.DateRange,
+            color = Color.Green,
+            subTitle = "이번달 쓴 돈",
+            title = "25,600원",
+        )
+    }
+}
+
+@Composable
+fun CardSection(
+    title: String,
+    content: @Composable ColumnScope.() -> Unit,
+) {
+    Card(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(16.dp),
+        colors = CardDefaults.cardColors(Color.White),
+    ) {
+        Column(
+            modifier = Modifier.padding(horizontal = 0.dp, vertical = 24.dp),
+        ) {
+            Text(
+                text = title,
+                color = Color.Black,
+                fontSize = 18.sp,
+                fontWeight = FontWeight.Bold,
+                modifier = Modifier.padding(horizontal = 20.dp, vertical = 0.dp),
+            )
+            content()
+        }
+    }
+}
+
+@Composable
+fun ItemSection(
+    icon: ImageVector,
+    color: Color,
+    subTitle: String?,
+    title: String,
+) {
+    Row(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(horizontal = 20.dp, vertical = 0.dp),
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.SpaceBetween,
+    ) {
+        Box(
+            modifier = Modifier
+                .size(42.dp)
+                .clip(shape = CircleShape)
+                .background(color),
+        ) {
+            Icon(
+                imageVector = icon,
+                contentDescription = null,
+                modifier = Modifier.align(Alignment.Center),
+            )
+        }
+
+        Column(
+            modifier = Modifier
+                .weight(1f)
+                .padding(horizontal = 24.dp, vertical = 0.dp),
+        ) {
+            if (subTitle == null) {
+                Text(
+                    text = title,
+                    fontSize = 16.sp,
+                    fontWeight = FontWeight.Bold,
+                )
+            } else {
+                Text(
+                    text = subTitle,
+                    fontSize = 12.sp,
+                    color = Color.Gray,
+                )
+                Text(
+                    text = title,
+                    fontSize = 16.sp,
+                    fontWeight = FontWeight.Bold,
+                    color = Color.Black,
+                )
+            }
```

Comment:
```suggestion
            if (subTitle == null) {
                Text(
                    text = title,
                    fontSize = 16.sp,
                    fontWeight = FontWeight.Bold,
                )
            } 
            Text(
                    text = title,
                    fontSize = 16.sp,
                    fontWeight = FontWeight.Bold,
                    color = Color.Black,
                )
           
```

이게 중복 코드도 제거하고 가독성도 더 좋은거같아요!